packages/framework/src/Component/ArrayUtils/ArrayHelper.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\ArrayUtils;
+
+class ArrayHelper
+{
+    /**
+     * @param array $array1
+     * @param array $array2
+     * @return bool
+     */
+    public function haveArraysDifferentValues(array $array1, array $array2): bool
```

Comment:
I really, really suffer that this class is a service... I think this is a great example of static method

Reply:
rewritten to static function :wink: